### DISCUSSION TITLE
Bookmark/Save Places with Category (Attractions/Restaurants..) and Status (Visited/To Go..)

### DISCUSSION THREAD

**Discussion by radh21301** [Freika/dawarich#222] [Ideas]
Building on the conversation with the dev on discord, here is my feature request.

This feature is all about keeping track of places we want to visit and updating their status once we've been there. I have been using Google maps for a long time now, but then tried multiple other apps (none were selfhosted).

Key components:

- "Places to Visit" list 📌: Let us create a wishlist of spots we're interested in checking out OR have been to already
- Categorization 🏷️ : Add tags like "Restaurants", "Museums", "Parks", etc.
  - Some tagging (tag groups and sub tags) options:
    - ✅ Status tracking (Visited/To Go) OR  
    - ⭐ ratings (Awesome/Good/Bad)
    - people can basically do whatever they want with it -> if we have tag groups and sub tags
- Notes 📝 : Option to jot down why we want to visit or any other relevant info
- Filtering 🔍 : Ability to view a filtered list of the places from those tags would be helpful
- Optional (Google maps link) 📍: I know we want to get away from google. But google maps is really helpful to get directions to that place, read reviews etc.

Some Inspiration: This is a snapshot of how it is in an app called [Proxi](https://www.proxi.co/). They have an extremely nice GUI and feels very refreshing to use -> better than google my maps in my opinion for some things. But it is not self hosted, and their privacy policy does not inspire confidence, especially for something involving location tracking.

------ **Categories and Tags** ------
<img width="329" alt="image" src="https://github.com/user-attachments/assets/c8632f21-1264-4dc8-b1f2-cf571c023f07">

------ **GUI to add a point** ------
<img width="326" alt="image" src="https://github.com/user-attachments/assets/8b0af74a-c370-4c08-8330-12ff7047b236">

------ **The Map Page** ------
<img width="1150" alt="image" src="https://github.com/user-attachments/assets/2ad13b79-7ed6-4bfb-a5e6-e0b7a3affe81">

**Comment by Freika** [comment]
This is planned (may be in some different form but still) although I can't provide you with any ETAs
